modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -42,7 +42,7 @@ When using CPU bursting, you must consider the following factors:
 +
 Regardless of what approach is chosen, make sure you run tests to validate performance.
 
-== Memory Allocation
+== Application Memory Allocation
```

Comment:
As the summary say, this is memory allocation for Mule application or API gateway, so please revert it back to Memory Allocation.

Reply:
Thank you - done!